src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IUpdaterResult.cs

```diff
@@ -5,6 +5,12 @@ namespace Microsoft.DotNet.UpgradeAssistant
 {
     public interface IUpdaterResult
     {
+        string RuleId { get; }
+
+        string RuleName { get; }
+
+        string FullDescription { get; }
```

Comment:
I was wondering about the HelpUri. Should I already include it here?

Reply:
I think that would make sense. Make it nullable and then it can just not be implemented